### PR TITLE
Improve output percentages

### LINKED ISSUE #60: Test success rate is improperly rounded
screen shot 2016-06-22 at 2 50 21 pm

### PR DESCRIPTION
**Pull request by avulfson17**
Ensures that 100% output will not occur with a failure.
Fixes #60

### CODE REVIEW

**Comment by orangejulius** [comment]
LGTM

**Comment by trescube** [comment]
:shipit:  we'll likely never have a single failed test out of 100,000 for rounding to matter

File: lib/percentageForDisplay.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+}
+
+function percentageForDisplay(total_tests, passed_tests){
+	if (total_tests === 0){
+		return 0;
+	}
+	return round(passed_tests * 100 / total_tests,2);
+}
+
+module.exports = percentageForDisplay;
```
Comment (orangejulius) [inline lib/percentageForDisplay.js]: aw man we have to fix the newlines! we'll figure out how to do it tomorrow

info and solution: https://robots.thoughtbot.com/no-newline-at-end-of-file

**Comment by trescube** [comment]
:shipit:

**Pull request merged**